src/nullxlsx.js

```diff
@@ -33,14 +33,16 @@ export class NullXlsx extends NullDownloader {
 	 * Create a spreadsheet from an array of arrays of data
 	 * @param {Array<Array<*>>} data Cell values
 	 * @param {string} name Name of sheet
+	 * @param {{[key: number]: 'left' | 'center' | 'right'}} [column_alignments] Column alignment settings
 	 * @return {NullXlsx} Returns itself for method chaining
 	 */
-	addSheetFromData(data, name) {
+	addSheetFromData(data, name, column_alignments) {
```

Comment:
It would be better to send in column_alignments as part of an options object, to allow room for any additional options.

```suggestion
	addSheetFromData(data, name, options) {
```